charts/fluentd-elasticsearch/templates/_helpers.tpl

```diff
@@ -23,6 +23,13 @@ If release name contains chart name it will be used as a full name.
 {{- end }}
 {{- end }}
 
+{{/*
+Returns the chart Release Name.
+*/}}
+{{- define "fluentd-elasticsearch.releaseName" -}}
+{{- .Release.Name | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
```

Comment:
why is this needed?
"Release.Name" is already available. 
See: https://helm.sh/docs/chart_template_guide/builtin_objects/

Reply:
I've had to use it because Release.Name is not directly available in the values.yaml file. It is only available in templating function.

The alternative was using the ["fluentd-elasticsearch.fullname"](https://github.com/kokuwaio/helm-charts/blob/main/charts/fluentd-elasticsearch/templates/_helpers.tpl#L13) alas the rendering of the PrometheusRule file was not the same as previously when .Release.Name was directly defined in the prometheusrule.yaml file.

The main issue is this rule: https://github.com/kokuwaio/helm-charts/blob/main/charts/fluentd-elasticsearch/templates/prometheusrule.yaml#L58
When using the templated fluentd-elasticsearch.fullname function, the rendered value was not the Release Name, this is why I had to include this function.

Reply:
Probably because of the scope. 
See. https://helm.sh/docs/chart_template_guide/variables/

But in tpl it works. I've changed it.